### DISCUSSION TITLE
cloudstack-management error

### DISCUSSION THREAD

**Discussion by khalikhu** [apache/cloudstack#9618] [Q&A] [answered]
##### ISSUE TYPE

 * Bug Report
 * Improvement Request
 * Enhancement Request
 * Feature Idea
 * Documentation Report
 * Other

##### COMPONENT NAME

~~~

~~~

##### CLOUDSTACK VERSION

~~~

~~~

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~
yum -y install cloudstack-management
Updating Subscription Management repositories.
cloudstack                                      215  B/s | 285  B     00:01    
Errors during downloading metadata for repository 'cloudstack':
  - Status code: 404 for http://download.cloudstack.org/centos/9/4.16/repodata/repomd.xml (IP: 000.111.222.3)
Error: Failed to download metadata for repo 'cloudstack': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by DaanHoogland** [comment, marked as answer]
@khalikhu 4.16 does not support el9. You'll have to use 4.18 or later (i would advice 4.19 atm)
